components/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./css/Header.module.css";
+
+const NavBar = () => {
+  return (
+      <nav className={styles.navbar}>
+        <h4 className={styles.brandlogo}>
+          <a href="/">
```

Comment:
Can you use the `Link` component? https://nextjs.org/docs/api-reference/next/link

Reply:
Just applied it to the latest commit